### DISCUSSION TITLE
Cable Sizes

### DISCUSSION THREAD

**Discussion by Nutty-69** [bitfieldlabs/afterglow#20] [Q&A] [answered]
What is the best cable sizes to use, the 2.54 connectors looks like they take up to 22AWG and the 3.96 take up to 18AWG. So 22AWG thick enough for use across all cables. (I am mainly interested in WPC & SYS11)

**Comment by bitfieldlabs** [comment, marked as answer]
All input wires only carry very small currents, so AWG 22 is fine. I use AWG 18 for the power supply (+18V, GND).
